core/src/test/scala/stryker4s/run/MutantRunnerTest.scala

```diff
@@ -95,5 +107,104 @@ class MutantRunnerTest extends Stryker4sIOSuite with MockitoIOSuite with LogMatc
         result.compileErrors shouldBe 1
       }
     }
+
+    it("should use static temp dir if it was requested") {
+      val fileCollectorMock = new TestFileResolver(Seq.empty)
+      val reporterMock = mock[Reporter]
+      whenF(reporterMock.onRunFinished(any[FinishedRunEvent])).thenReturn(())
+      when(reporterMock.mutantTested).thenReturn(_.drain)
+      val mutant = Mutant(MutantId(1), q"0", q"zero", EmptyString)
+
+      val testRunner = { (path: Path) =>
+        // Static temp dir is used.
+        path.toString should endWith("stryker4s-tmpDir")
+        TestRunnerStub.withResults(Killed(mutant))(path)
+      }
+
+      val sut = new MutantRunner(testRunner, fileCollectorMock, reporterMock)(staticTmpDirConfig, testLogger)
+      val file = FileUtil.getResource("scalaFiles/simpleFile.scala")
+      val mutants = Seq(mutant)
+      val mutatedFile = MutatedFile(file, q"def foo = 4", mutants, Seq.empty, 0)
+
+      sut(_ => IO.pure(List(mutatedFile))).asserting { result =>
+        val captor = ArgCaptor[FinishedRunEvent]
+        verify(reporterMock, times(1)).onRunFinished(captor.capture)
+        val runReport = captor.value.report.files.loneElement
+
+        "Setting up mutated environment..." shouldBe loggedAsInfo
```

Comment:
Is adding such simple assertions here ok? Or test for even less, as mutation are checked in another test case?

Reply:
To keep the test clean, it's best to only test for the changes that this test has compared to the others. So since logging is already covered in other tests you don't need to assert for it again

Reply:
Removed some assertions. Should I remove even more?